src/cli/commands/test/mod.rs

```diff
@@ -92,6 +94,111 @@ impl From<(String, TestStatus)> for TestResult {
 	}
 }
 
+/// structure used to store a Mocked function in a cairo test entrypoint
+/// fn_name: the name of the function to mock in the cairo file
+/// fn_args: the list of parameters to replace the mock function call
+#[derive(Debug)]
+pub struct MockEntry<'a> {
+	pub fn_name: &'a str,
+	pub fn_args: &'a str,
```

Comment:
We don't need to know what the user intends to mock the call with at this point.
It's handle elsewhere in the execution